kernel/component-manager/src/main/bundle/org/sakaiproject/config/kernel.properties

```diff
@@ -26,6 +26,8 @@ helpPath=/help
 # Use /tool for the new dispatcher
 toolPath=/portal/tool
 
+# Default favicon
+portal.favicon.url=/library/icon/favicon.ico
```

Comment:
This is a fine change but i kind of lean towards an empty default .... WDYT?

Reply:
I did test that it works if there's no default, but I feel like for the nightlies we'll want to set a default. I also changed the default to the [new jewel icon](https://raw.githubusercontent.com/jonespm/sakai/SAK-43903/library/src/webapp/icon/favicon.ico). I think it looks better than having the standard "non-icon" and it also provides a little marketing for schools that don't want to change it.

The default for Canvas is the Instructure icon and even though there is a UI to change it, I spot checked a few schools and only half of the ones I looked at actually did change it.